views/lexicon/tabs/progress_bars.html.twig

```diff
@@ -216,5 +216,32 @@
         </a>
 
     </div>
+    <br/>
+    <h2 class="heading">Stepped Progress Bar</h2>
+
+    <div class="stepped-progressbar">
+        <div class="step"><span class="step-box current">1</span></div>
+        <div class="step"><span class="step-box">2</span></div>
+        <div class="step"><span class="step-box">3</span></div>
+        <div class="step last"><span class="step-box"><i class="icon-flag-checkered"></i></span></div>
+    </div><br/>
+    <div class="stepped-progressbar">
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step"><span class="step-box current">2</span></div>
+        <div class="step"><span class="step-box">3</span></div>
+        <div class="step last"><span class="step-box"><i class="icon-flag-checkered"></i></span></div>
+    </div><br/>
+    <div class="stepped-progressbar">
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step"><span class="step-box current">3</span></div>
+        <div class="step last"><span class="step-box"><i class="icon-flag-checkered"></i></span></div>
+    </div><br/>
+    <div class="stepped-progressbar">
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step successful"><span class="step-box completed"><i class="icon-ok"></i></span></div>
+        <div class="step last"><span class="step-box final"><i class="icon-flag-checkered"></i></span></div>
+    </div><br/><br/>
```

Comment:
Margin rather than `<br/><br/>`?

Reply:
Same as the comment above

Reply:
I don't see why they are needed then? Can we remove as if they aren't needed when used having them in the lexicon and doc will add confusion

Reply:
"<br/>" substituted with margin-bottom